presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -912,7 +912,7 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
                     "create an encrypted table without partitions");
         }
 
-        Map<String, String> tableProperties = getEmptyTableProperties(tableMetadata, new HdfsContext(session, schemaName, tableName), hiveStorageFormat, tableEncryptionProperties);
+        Map<String, String> tableProperties = getEmptyTableProperties(tableMetadata, new HdfsContext(session, schemaName, tableName, true), hiveStorageFormat, tableEncryptionProperties);
```

Comment:
Let's specify the table path here as well. It can be moved down below, where the path is available.